components/nds-card/src/Card.js

```diff
@@ -28,38 +28,51 @@ export const Card = props => {
 		headingText,
 		headingElementType,
 		link,
+		image,
+		children,
 		elementType: ContainerType = "article",
 		...rest
 	} = props;
 	const headerProps = { headingText, headingElementType, link };
 	return (
 		<ContainerType className="card" {...rest}>
-			<header className="card__header">
-				<CardHeader {...headerProps} />
-			</header>
-			{metadata && metadata.length && (
-				<dl className="card__metadata">
-					{metadata.map((item, idx) => {
-						if (!item.value) return null;
-						return (
-							<div key={`item${idx}`} className="card__metadatum">
-								{item.label && (
-									<dt className="visually-hidden">{item.label}</dt>
-								)}
-								<dd>{item.value}</dd>
-							</div>
-						);
-					})}
-				</dl>
+			{image && image.src && (
+				<img className="card__image" src={image.src} alt={image.alt} />
 			)}
+			<div className="card__text">
+				<header className="card__header">
+					<CardHeader {...headerProps} />
+				</header>
+				{children && <div className="card__summary">{children}</div>}
+				{metadata && metadata.length && (
+					<dl className="card__metadata">
+						{metadata.map((item, idx) => {
+							if (!item.value) return null;
+							return (
+								<div key={`item${idx}`} className="card__metadatum">
+									{item.label && (
+										<dt className="visually-hidden">{item.label}</dt>
+									)}
+									<dd>{item.value}</dd>
+								</div>
+							);
+						})}
+					</dl>
+				)}
+			</div>
 		</ContainerType>
 	);
 };
 
 Card.propTypes = {
+	children: PropTypes.node,
 	elementType: PropTypes.elementType,
 	headingElementType: PropTypes.elementType,
 	headingText: PropTypes.node.isRequired,
+	image: PropTypes.shape({
```

Comment:
Maybe it's better to allow passing in a whole img tag, to be able to support things like Img from gatsby:

```
<Img
      fixed={data.file.childImageSharp.fixed}
      alt=""
    />
```

I can't see how we'd do it with a props object rather than taking an element

Reply:
Sure - good point - will sort this.

Reply:
@ediblecode As I understand, `gatsby-image` will pass the img tag directly without additional markup, so do you think this is safe for this use case? whatever gets passed gets the required className mixed with any others supplied - https://github.com/nice-digital/nice-design-system/blob/d7c50a6d58315773180b37a46dc37bf54da37069/components/nds-card/src/Card.js#L40-L43

Reply:
I think it's more complicated than that - it renders a wrapper span (or two). BUT according to the [gatsby-image docs](https://www.gatsbyjs.org/packages/gatsby-image/#gatsby-image-props passing className), className gets added to the wrapper element so we should be all good

Reply:
Hmm I think we want `.card__image` directly on the image tag in this case. Unless we make it the responsibility of the user to pass the correct class to the image before it's passed in?

Reply:
Add class at any depth? https://github.com/nice-digital/nice-design-system/blob/3c2fd6e03f7c235388d0d76aad7b148acc44250d/components/nds-card/src/Card.js#L27-L59

Reply:
So I was massively overthinking that!